### ISSUE TITLE
Cannot use ImageFilter over Unity Widget

### ISSUE THREAD

**Issue by utkarshgupta2504** [juicycleff/flutter-unity-view-widget#748]
**Describe the bug**
Image Filter gets ignored atop of this UnityWidget.
Using ImageFilter.blur for giving an aesthetic container for controlling the widget.
Works on the normal [camera widget](https://pub.dev/packages/camera) provided by flutter flawlessly

**To Reproduce**
1. Initialize the Unity Application (Export and setting up)
2. Using a `Stack` widget, place `UnityWidget`, and then a `ClipRRect` with `BackdropFilter(filter: ImageFilter.blur())`

**Expected behavior**
The screen should have a blurred container on top of the UnityWidget

**Screenshots**
![Current (With UnityWidget)](https://user-images.githubusercontent.com/52123629/213158908-eea38012-c4da-41e1-a341-d6519c5a3d54.jpeg)
![Expected (With Camera Widget)](https://user-images.githubusercontent.com/52123629/213158922-2ebe0aab-cccf-4819-92da-0a68254aa864.jpeg)

**Unity:**
 - OS: MacOS 13
 - Version: v2021.3.11f1

**Smartphone:**
 - Device: Samsung A52s
 - OS: Android
 - Version: 12

**Comment by sundong212** [comment]
Same here.

What's your flutter version? I'm on 2.10.4, which is relatively outdated. I'm gonna try to upgrade to the latest. Since I found this 
https://github.com/flutter/flutter/issues/43902 

They said it's been fixed. But I doubt if this can fix the unity issue. But internally, both of them are having the same issue because of the different renderer engine.

**Comment by timbotimbo** [comment]
Just did a quick test on android with Flutter 3.3.8.

Just like the rounded corners, the blur works with `useAndroidViewSurface: false` (left).
Both are broken for `useAndroidViewSurface: true` (right), which has quite a few bugs like this.

<img src="https://user-images.githubusercontent.com/11444698/214401545-2b9137d3-db08-4be8-a764-80ca22205185.jpg" width=25%><img src="https://user-images.githubusercontent.com/11444698/214401555-66f6f699-2621-4784-8509-353f6e83f07c.jpg" width=25%>
<details><summary>Code snippet</summary>

```dart
 Card(
  color: Colors.transparent,
  elevation: 0,
  child: ClipRRect(
	child: BackdropFilter(
	  filter: ImageFilter.blur(sigmaX: 5, sigmaY: 5),
	  child: Column(
		children: <Widget>[
		  Padding(
			padding: const EdgeInsets.only(top: 20),
			child: Text("Rotation speed:"),
		  ),
		  Slider(
```
</details>

I believe you switched to `useAndroidViewSurface: true` because of touch input issues.
The only main drawback of using false is that it is broken with Unity's new input package, you'll need to use the legacy input system or a mix of both.

**Comment by utkarshgupta2504** [comment]
> Just did a quick test on android with Flutter 3.3.8.
> 
> Just like the rounded corners, the blur works with `useAndroidViewSurface: false` (left).
> Both are broken for `useAndroidViewSurface: true` (right), which has quite a few bugs like this.
> 
> I believe you switched to `useAndroidViewSurface: true` because of touch input issues.
> The main drawback of using `false` is that touch input is broken with Unity's new input package, you'll need to use the legacy input system or a mix of both.
> Besides touch, this setup is more reliable in my experience.

Yeah have been using the new input package, but that is an interesting insight, will try it put with my AR application, if it works, I will update this comment, thank you so much!

**Comment by sundong212** [comment]
> Just did a quick test on android with Flutter 3.3.8.
> 
> Just like the rounded corners, the blur works with `useAndroidViewSurface: false` (left). Both are broken for `useAndroidViewSurface: true` (right), which has quite a few bugs like this.
> 
> <img alt="" width="25%" src="https://user-images.githubusercontent.com/11444698/214401545-2b9137d3-db08-4be8-a764-80ca22205185.jpg"><img alt="" width="25%" src="https://user-images.githubusercontent.com/11444698/214401555-66f6f699-2621-4784-8509-353f6e83f07c.jpg">
> 
> Code snippet
> I believe you switched to `useAndroidViewSurface: true` because of touch input issues. The main drawback of using `false` is that touch input is broken with Unity's new input package, you'll need to use the legacy input system or a mix of both. Besides touch, this setup is more reliable in my experience.
> 
> According to the changelog, this should work on iOS since Flutter 3.7, but I haven't tried it yet.

Still not working on iOS by Flutter 3.7

**Comment by timbotimbo** [comment]
I see blur failing for `google_maps_flutter` too when you enable Hybrid composition mode.
```
if (mapsImplementation is GoogleMapsFlutterAndroid) {
    mapsImplementation.useAndroidViewSurface = true;
  }
```

I assume this is a general bug in Flutter with this type of Platform View.

**Comment by ShakeelKhalid1913** [comment]
hey if you are able to run the project can you help me. i cant find any source for flutter 3.7.x
